projects/scion/microfrontend-platform/src/lib/client/messaging/intent-client.ts

```diff
@@ -128,20 +128,13 @@ export abstract class IntentClient {
    *  });
    * ```
    *
-   * @param  selector - Allows filtering intents. The qualifier allows using wildcards (such as `*` or `?`) to match multiple intents simultaneously.\
+   * @param  selector - Allows filtering intents. The qualifier allows using the asterisk wildcard character (`*`) to match multiple intents simultaneously.\
    *         Note that the passed filter is only a filter for intents the application is qualified for, i.e., provides a fulfilling capability visible
    *         to the sender.
    *         <p>
    *         <ul>
-   *           <li>**Asterisk wildcard character (`*`):**\
```

Comment:
I would rephrase the parameter's description, as following:

```code
@param  selector - Allows filtering intents. Note that the passed filter is only a filter for intents the application is qualified for, i.e., provides a fulfilling capability visible to the sender.
        In the qualifier of the filter, you can use the asterisk wildcard character (`*`) to match multiple intents simultaneously.
        - Asterisk wildcard character (*)
          Matches intents with such a qualifier property no matter of its value (except `null` or `undefined`). Use it like this: `{property: '*'}`.
        - Partial wildcard (**)
          Matches intents even if having additional properties. Use it like this: `{'*': '*'}`.
```

Reply:
done